tests/test_log.py

```diff
@@ -34,7 +34,10 @@
 
 from six.moves import reload_module
 
-
+# We explicitly create a logger which is a child of the tuf hierarchy,
+# instead of using the standard getLogger(__name__) pattern, because the
+# tests are not part of the tuf hierarchy and we are testing functionality
+# of the tuf package explicitly enabled on the tuf hierarchy
 logger = logging.getLogger('tuf.test_log')
```

Comment:
```suggestion
logger = logging.getLogger(__name__)
```

Reply:
Per the comment above I explicitly didn't change to `__name__` here because that gives us a logger names "test_log", which isn't part of the "tuf" logger hierarchy and therefore we can not test the logging functionality in `tuf.log` which only affects the "tuf" logger and its children.

Reply:
Got it, somehow missed that! Curious, is this necessary for the tests to work?

Reply:
It is necessary for the tests to work.

Functions in `tuf.log` change logger configuration settings on the "tuf" logger and its children. Therefore if we create a logger in the test named "test_log" (via `getLogger(__name__)`) the `tuf.log` calls don't affect the logger we created (as it's not a member of the "tuf" logger hierarchy) and the tests fail:
```
======================================================================
FAIL: test_set_log_level (test_log.TestLog)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/joshuagl/Projects/tuf/tests/test_log.py", line 62, in test_set_log_level
    self.assertTrue(logger.isEnabledFor(logging.DEBUG))
AssertionError: False is not true
```